docs/current_docs/api/module-structure.mdx

```diff
@@ -263,6 +280,51 @@ class MyModule
 }
 ```
 
+</TabItem>
+<TabItem value="Java">
+    The Dagger module's code in Java can be split into multiple classes, in multiple files. A few constraints apply:
+
+    - the main object must be named by the name of the module in PascalCase. For instance if the module is `my-module`, the main class must be named `MyModule`.
```

Comment:
```suggestion
    - The main object must use the same name as the module, in PascalCase. For instance, if the module name is `my-module`, the main class must be named `MyModule`.
```

Which is correct: main object or main class? They are both used in the above sentence.

Reply:
The grammar/copy was already updated.

I just pushed a small change to (I hope) make it more understandable regarding the Dagger object and the Java class.